lib/WP_Auth0_Options_Generic.php

```diff
@@ -174,16 +169,35 @@ public function set( $key, $value, $should_update = true ) {
 		return $this->update_all();
 	}
 
+	/**
+	 * Remove a setting from the options array in memory.
+	 *
+	 * @param string $key - Option key name to remove.
+	 */
+	public function remove( $key ) {
+		$options = $this->get_options();
```

Comment:
you shouldn't need to call this method before asserting the `$key` is a constant. Let's move this line below that check and save that call for when it's actually required.